src/backend/base/langflow/components/cleanlab/cleanlab_rag_evaluator.py

```
@@ -0,0 +1,254 @@
+class CleanlabRAGEvaluator(Component):
+    """A component that evaluates the quality of RAG (Retrieval-Augmented Generation) outputs using Cleanlab.
+
+    This component takes a query, retrieved context, and generated response from a RAG pipeline,
+    and uses Cleanlab's evaluation algorithms to assess various aspects of the RAG system's performance.
+
+    The component can evaluate:
+    - Overall trustworthiness of the LLM generated response
+    - Context sufficiency (whether the retrieved context contains information needed to answer the query)
+    - Response groundedness (whether the response is supported directly by the context)
+    - Response helpfulness (whether the response effectively addresses the user's query)
+    - Query ease (whether the user query seems easy for an AI system to properly handle, useful to diagnose 
```

Ruff (0.11.9)

Trailing whitespace

Remove trailing whitespace

(W291)